pulpcore/cli/common/__init__.py

```diff
@@ -100,6 +100,7 @@ def _config_callback(ctx: click.Context, param: Any, value: Optional[str]) -> No
     help=_("Start tasks in the background instead of awaiting them"),
 )
 @click.option("--refresh-api", is_flag=True, help=_("Invalidate cached API docs"))
+@click.option("--cid", help=_("Logging CID to send on requests"))
```

Comment:
Can we give the user a little help, something like this:
```suggestion
@click.option("--cid", help=_("Logging CID to send on requests (note: server configuration may require a valid GUID and ignore CIDs that aren't)"))
```